app/javascript/tagging/components/Tagging/Tagging.js

```diff
@@ -41,9 +41,12 @@ class Tagging extends React.Component {
   findSelectedTag = (selectedTagCategory = { id: undefined }) =>
     this.props.tags.find(tag => tag.id === selectedTagCategory.id);
 
-  isMulti = selectedTagCategory =>
-    this.findSelectedTag(selectedTagCategory) &&
-    this.findSelectedTag(selectedTagCategory).singleValue === false;
+  isMulti = (selectedTagCategory) => {
+    if (this.findSelectedTag(selectedTagCategory) && this.findSelectedTag(selectedTagCategory).singleValue) {
+      return !this.findSelectedTag(selectedTagCategory).singleValue;
```

Comment:
is there a way to DRY this up?  Something like:

```suggestion
    selectedTag = this.findSelectedTag(selectedTagCategory);
    if (selectedTag && selectedTag.singleValue) {
      return !selectedTag.singleValue;
```

Reply:
And now that I'm reading that, how does this even work?  If `singleValue` is `false`, then `selectedTag && selectedTag.singleValue` will return false, which means it falls through to line 48 and returns `false`, but it _should_ return true.

Reply:
I will check about testcases.

Reply:
@Fryguy Fixed code and added testcases.